shark_turbine/kernel/wave/expansion.py

```diff
@@ -12,6 +12,7 @@
 from .._support.indexing import IndexingContext
 from ...support.logging import get_logger
 from .._support.tracing import CapturedTrace
+from .._support.indexing import index_symbol
 from .indexing import IndexSequence
```

Comment:
Sorry, regarding a previous PR:
Why is `IndexSequence` not part of `.._support.indexing`. 
Also its import in `wave_ops.py` is missing.

Reply:
Good question. Hadn't look much into that code but seems like it could fit there. Will move and add import into wave_ops.py